### ISSUE TITLE
waveshare picoRES-TOUCHLCD-3.5

### ISSUE THREAD

**Issue by mrbbp** [bablokb/circuitpython-examples#1]
hello
the code does not work without the patch.
how to patch the busio source?
have to take the source code of circuitpython 9, patch and compile a specific version?
is there an other way?
i'm not so tech.
regards

**Comment by bablokb** [comment]
Yes, that is what you have to do. There is already a patched version in my fork of CircuitPython: https://github.com/bablokb/circuitpython/tree/ili9488-ws3.5. This is not the most current version of CP, but that is the one I am using.

To compile, please follow the instructions in https://github.com/bablokb/circuitpython/blob/ili9488-ws3.5/.devcontainer/Readme.md
